x2gbfs/providers/cantamen.py

```diff
@@ -351,6 +351,11 @@ def load_vehicles(self, default_last_reported: int) -> Tuple[Dict, Dict]:
                 )
                 vehicles[gbfs_vehicle['bike_id']] = gbfs_vehicle
                 vehicle_types[vehicle_type['vehicle_type_id']] = vehicle_type
+            except ValueError as ex:
+                logger.warning(
+                    f'Could not extract vehicle/vehicle_type for bookee {bookee_id} due to ValueError: {ex.args}',
+                    exc_info=False,
```

Comment:
For warning, exc_info=False is default for logging.warning not explicitly required. I leave it up to you, if you want to leave it in here to emphasize that the stacktrace is not logged